README.md

```diff
@@ -2,391 +1,310 @@
+![image-concept-overview.png](docs/images/image-concept-overview.png)
 
-## Installation
-Installation can be done in three ways:
+It provides a number of convenience features for developers, that make it simple to do the following essential tasks:
+* Creating signed security requests for API initialization, and
+* Interacting with the Data API.
 
-1.) Download the jar in directory learnosity-sdk-java/Dist/All/ and add to your build path. All the required libraries are included in this jar.
+For example, the SDK helps with creating a signed request for Learnosity:
 
-2.) Download the jar in learnosity-sdk-java/Dist/Light/ and add to your build path. This requires the external libraries which can be found in learnosity-sdk-java/Dist/Light/RequiredJars
+![image-signed-request-creation.png](docs/images/image-signed-request-creation.png)
 
-3.) Download the source code and integrate into your project. This requires the external libraries which can be found in learnosity-sdk-java/Dist/Light/RequiredJars
+Once the SDK has created the signed request for you, your app sends that on to an API in the Learnosity cloud, which then retrieves the assessment you are asking for, as seen in the diagram below:
 
-4.) To use as a dependency in a maven project see below 'Using with a file based repository'
+![image-assessment-retrieval.png](docs/images/image-assessment-retrieval.png)
 
-## Test
+This scenario is what you can see running in the Quick start guide example ([see below](#quick-start-guide)).
 
-In order to check that you've added the required code correctly, you can download src/LearnositySdk/Test and try to run the test file. If it runs successfully, all should be fine.
+There's more features, besides. See the detailed list of SDK features on the [reference page](REFERENCE.md).
 
-The Test.java class also gives some examples on how to use the sdk.
+[(Back to top)](#table-of-contents)
 
-## Usage
+## Requirements
+1. Runtime libraries for Java installed. We've used Eclipse Adoptium/Temurin 11 LTS (previously known as 'AdoptOpenJDK'). ([instructions](https://projects.eclipse.org/projects/adoptium.temurin/downloads))
 
-### Init
+2. Maven installed ([instructions](https://maven.apache.org/download.cgi)). We tested on Maven 3.8.4.
 
-The Init class is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+3. Docker installed (required for the example tutorial on this page). ([instructions](https://www.docker.com/get-started)). We tested on Docker version 3.x.
 
-The Init constructor takes either 3 arguments:
+Note: just looking for a .WAR file? Look in the Maven target directory.
 
- * String  service type
- * Object  security details (**no secret**)
- * String  secret
+Not using Java? See the [SDKs for other languages](https://help.learnosity.com/hc/en-us/sections/360000194318-Server-side-development-SDKs).
 
-or 4 arguments:
+### **Supported Java Versions**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
### **Supported Java Versions**

```